examples/job_stac.ipynb

```diff
@@ -0,0 +1,135 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.5 64-bit",
+   "metadata": {
+    "interpreter": {
+     "hash": "31862cc71836a94b0e0781803a3648767fc4cb197cc35bade0ddf231ddce7d7c"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "## Harmony Py Library\n",
+    "### Job Results STAC data"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append('..')\n",
+    "from harmony import BBox, Client, Collection, Request\n"
+   ]
+  },
+  {
+   "source": [
+    "First, let's get a job processing in Harmony."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "harmony_client = Client()  # assumes .netrc usage\n",
+    "\n",
+    "collection = Collection(id='C1234088182-EEDTEST')\n",
+    "request = Request(\n",
+    "    collection=collection,\n",
+    "    spatial=BBox(-165, 52, -140, 77)\n",
+    ")\n",
+    "\n",
+    "job_id = harmony_client.submit(request)\n",
+    "job_id\n"
+   ]
+  },
+  {
+   "source": [
+    "Harmony-py can return the STAC Catalog URL for a completed job."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stac_catalog_url = harmony_client.stac_catalog_url(job_id, show_progress=True)\n"
+   ]
+  },
+  {
+   "source": [
+    "Following the directions for PySTAC (https://pystac.readthedocs.io/en/latest/quickstart.html), we can hook our harmony-py client into STAC_IO."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from urllib.parse import urlparse\n",
+    "import requests\n",
+    "from pystac import STAC_IO\n",
+    "\n",
+    "def requests_read_method(uri):\n",
+    "    parsed = urlparse(uri)\n",
+    "    if parsed.hostname.startswith('harmony.'):\n",
+    "        return harmony_client.read_text(uri)\n",
+    "    else:\n",
+    "        return STAC_IO.default_read_text_method(uri)\n",
+    "\n",
+    "STAC_IO.read_text_method = requests_read_method"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pystac import Catalog\n",
+    "\n",
+    "cat = Catalog.from_file(stac_catalog_url)"
```

Comment:
This notebook doesn't have enough output to really show that it worked.  Try the following in a cell:

```python
print(cat.title)
for item in cat.get_all_items():
    print(item.datetime, [asset.href for asset in item.assets.values()])
```

Reply:
I think you wrote more of this card than I did  : ) I definitely appreciate the help, though.